cmd/osv-scanner/main.go

```diff
@@ -97,16 +97,19 @@ func run(args []string, stdout, stderr io.Writer) int {
 				Usage:   "check subdirectories",
 				Value:   false,
 			},
-			&cli.BoolFlag{
```

Comment:
question: should we keep this for a release or two and print a deprecation warning about it going away in a non-major?

I know we don't _have_ to, but it's pretty easy and only costs a few bytes in the binary

Reply:
Thanks for pointing out. It definitely will be more user friendly.